assets/_scss/elements/_typography.scss

```diff
@@ -98,6 +98,17 @@ h6 {
   @include h6();
 }
 
+// Remove user agent styles
+
+i, 
+cite, 
+em, 
```

Comment:
I wouldn't think we should actually set em font-style to normal, I can file an issue.